### DISCUSSION TITLE
A Question about relations

### DISCUSSION THREAD

**Discussion by Lucas-Gruber** [directus/directus#17568] [Archive]
Hey,
i have a question about relations.

I have a collection for employees and one for cars.
Now i want employees to be able to add cars they own to their listing. However, it must also be the case that an employee can add a car several times. How can I best implement this?

**Comment by jtlindsey** [comment]
You could create a many to many field (lets call it `cars`) in employees collection that points to cars collection and then for the employees `cars` field settings go to interface > Filter and set `User Created -> ID Equals $CURRENT_USER`

That will enable the following:
- employee can add 0 to many cars
- employee can only add cars they created

As far as _"an employee can add a car several times"_, you'll see an option to "Allow duplicates" on the interface page.

I don't know if it's the _best_ way but it is one way.

Relevant docs:
https://docs.directus.io/configuration/data-model.html
https://docs.directus.io/reference/filter-rules.html

**Reply by Lucas-Gruber** [reply]
Okay thank you,
how can i achieve that employees can give their cars a license plate?
